floater/generators.py

```diff
@@ -40,6 +40,14 @@ def __init__(self, xlim, ylim, dx=1., dy=1.):
             grid spacing in x direction
         dy : float
             grid spacing in y direction
+    model_grid : dictionary
+        the following key value pairs are expected
+            'land_mask': np.ndarray of bools
+                2d array of dimensions in C order: shape==(len(lat), len(lon))
```

Comment:
I also modified the dimension order of land_mask.

In python, the convention is to use C-order ([row-major order](https://en.wikipedia.org/wiki/Row-_and_column-major_order)) for multidimensional arrays.

You will have to change your user code accordingly.